sdk/cargo-build-bpf/tests/crates/noop/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "noop"
+version = "1.8.0"
+description = "Solana BPF test program written in Rust"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+repository = "https://github.com/solana-labs/solana"
+license = "Apache-2.0"
+homepage = "https://solana.com/"
+edition = "2018"
```

Let's add a `publish = false` to ensure this'll never get published to crates.io by accident by CI
```suggestion
edition = "2018"
publish = false
```